experiments/ssl4eo/conf/tm_toa_cdl.yaml

```diff
@@ -0,0 +1,34 @@
+module:
+  _target_: torchgeo.trainers.SemanticSegmentationTask
+  loss: "ce"
+  model: "unet"
+  backbone: "resnet18"
+  weights: true
+  learning_rate: 1e-4
+  learning_rate_schedule_patience: 6
+  in_channels: 7
+  num_classes: 134
+  num_filters: 64
+  ignore_index: 0
+  weight_decay: 0
+
+datamodule:
+  _target_: torchgeo.datamodules.SSL4EOLBenchmarkDataModule
+  root: path/to/ds
+  input_sensor: tm_toa
+  mask_product: cdl
+  batch_size: 256
+  num_workers: 4
+
+trainer:
+  _target_: lightning.pytorch.Trainer
+  accelerator: cpu
+  min_epochs: 50
+  max_epochs: 100
+
+program:
+  seed: 0
+  output_dir: output/tm_toa_cdl/
+  log_dir: logs/tm_toa_cdl/
+  overwrite: True
+  experiment_name: test_downstream_tm_toa
```

Comment:
What text editor do you use? There should be a way to prevent this line ending bug